scripts/contentScript.js

```diff
@@ -245,106 +245,165 @@ function addCssElementToBitbucket(highlightedPRIds, userId) {
-function getHighlightedPR(repoOwner, repoName, userId) {
-	chrome.storage.sync.get(["backendUrl"]).then(async ({ backendUrl }) => {
-		const body = {
-			"repo_owner": repoOwner,
-			"repo_name": repoName,
-			"user_id": userId,
-			"is_github": true
-		}
-		const url = `${backendUrl}/relevance/pr`;
-		const highlightedPRIds = await apiCall(url, body);
-		if (highlightedPRIds) {
-			for (const priorityLevel in highlightedPRIds) {
-				for (const prNumber in highlightedPRIds[priorityLevel]) {
-					addingCssElementToGithub(prNumber, keyToLabel[priorityLevel], highlightedPRIds[priorityLevel][prNumber]['num_files_changed'])
-				}
+function getHighlightedPR(highlightedPRIds) {
+	if (highlightedPRIds) {
+		for (const priorityLevel in highlightedPRIds) {
+			for (const prNumber in highlightedPRIds[priorityLevel]) {
+				addingCssElementToGithub(prNumber, keyToLabel[priorityLevel], highlightedPRIds[priorityLevel][prNumber]['num_files_changed'])
 			}
 		}
-	})
+	}
 };
 
 // adding favButton
 async function showFloatingActionButton(orgName, orgRepo, userId, websiteUrl) {
-
 	const trackedRepoList = await getTrackedRepos(orgName, userId);
-
 	if (!trackedRepoList.includes(orgRepo)) {
 		createElement("add", websiteUrl);
 	}
 }
 
 // showing the important files in a pr
-async function showImpFileInPr(repoOwner, repoName, userId, pr_number) {
-	chrome.storage.sync.get(["backendUrl"]).then(async ({ backendUrl }) => {
-		const body = {
-			"repo_owner": repoOwner,
-			"repo_name": repoName,
-			"user_id": userId,
-			"pr_number": pr_number,
-			"is_github": true
-		}
-		const url = `${backendUrl}/relevance/pr/files`;
-		let response = await apiCall(url, body);
-		if ("relevant" in response) {
-			const encryptedFileNames = new Set(response['relevant']);
-			const fileNav = document.querySelector('[aria-label="File Tree Navigation"]');
-			if (!fileNav) return;
-			const fileList = Array.from(fileNav.getElementsByTagName('li'));
-			fileList.forEach(async (item) => {
-				let elements = item.getElementsByClassName('ActionList-item-label');
-				if (elements.length == 1) {
-					let filename = elements[0].innerHTML.trim();
-					const hashedFilename = await sha256(filename);
-					if (encryptedFileNames.has(hashedFilename)) {
-						item.style.backgroundColor = '#7a7e00';
+async function showImpFileInPr(response) {
+	if ("relevant" in response) {
+		const encryptedFileNames = new Set(response['relevant']);
+		const fileNav = document.querySelector('[aria-label="File Tree Navigation"]');
+		if (!fileNav) return;
+		const fileList = Array.from(fileNav.getElementsByTagName('li'));
+		fileList.forEach(async (item) => {
+			let elements = item.getElementsByClassName('ActionList-item-label');
+			if (elements.length == 1) {
+				let filename = elements[0].innerHTML.trim();
+				const hashedFilename = await sha256(filename);
+				if (encryptedFileNames.has(hashedFilename)) {
+					item.style.backgroundColor = '#7a7e00';
+				}
+			}
+		})
+	}
+}
```

Comment:
Does this change fix `highlighting files in a PR tree` problem?

Reply:
Yes